src/Client/components/ui/TrueTestLogo.tsx

```
@@ -1,9 +1,18 @@
+    textsz?: string ;
+  }
+
+export default function Logo({ size, textsz }: LogoProps){
+    const svgSize = size ?? 28;
+    const width =  svgSize;
+    const height =  svgSize;
+    const text= textsz ?? 'text-xl'
     return(
         <div className="flex items-center gap-1 ml-2 ">
-          <svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <svg width={width} height={height} viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)